api/status.py

```diff
@@ -0,0 +1,8 @@
+from flask import Blueprint, Response
+
+status_blueprint = Blueprint('status', __name__)
+
+
+@status_blueprint.route('/status', methods=['GET'])
+def validate_schema():
+    return Response(status=200)
```

Comment:
It's probably worth adding the application version to the status response as we do in https://github.com/ONSdigital/eq-survey-runner/blob/master/app/setup.py#L265